portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ul.topnav {
```

Comment:
Avoid qualifying ID and class names with type selectors unless it's necessary (https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors)